tests/testthat/examples/Test_2023_1/output/traits.csv

```diff
@@ -1,123 +1,123 @@
 dataset_id,taxon_name,observation_id,trait_name,value,unit,entity_type,value_type,basis_of_value,replicates,basis_of_record,life_stage,population_id,individual_id,temporal_id,source_id,location_id,entity_context_id,plot_id,treatment_id,collection_date,measurement_remarks,method_id,original_name
-Test_2023_1,Acacia celsa,001,flowering_time,nyynnnnnnnnn,,individual,mode,expert_score,3,field,adult,01,01,02,,01,03,01,01,2004/2004,from Falster_2005_1,07,Acacia celsa
```

Comment:
Can you confirm that all these changes I outputs are what you intended? If so, happy for you to proceed

Reply:
It looks good to me because I added a new `entity_context` called "plant diameter" which should have changed all the `entity_context_id` numbers, which it looks like has happened.